extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -238,7 +237,7 @@ private static String minusParquetSuffix(@NotNull final String s) {
     }
 
     /**
-     * Generates a grouping file path relative to the table destination file path.
+     * Generates the index file path relative to the table destination file path.
```

Comment:
Incomplete. Still talks about grouping files, and uses "g" as the example column name.